src/app/workspace/[workspaceId]/header.tsx

```
@@ -28,188 +29,210 @@ import { cn } from "@/lib/utils";
+                      </div>
+                    )}
+                  </button>
+                ))}
+
+                <button
+                  className="flex w-full cursor-pointer items-center gap-x-4 rounded-[10px] border border-dashed bg-card/50 px-4 py-3 hover:bg-accent/10 transition-standard hover:translate-x-1 group mt-2"
+                  onClick={() => {
+                    setSwitchOpen(false); // updated here
+                    setCreateOpen(true);
+                  }}
+                >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)